packages/google_mlkit_document_scanner/android/build.gradle

```diff
@@ -0,0 +1,58 @@
+group 'com.google_mlkit_document_scanner'
+version '1.0'
+
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:7.3.0'
+    }
+}
+
+rootProject.allprojects {
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+    if (project.android.hasProperty("namespace")) {
+        namespace 'com.google_mlkit_document_scanner'
+    }
+
+    compileSdkVersion 33
```

Comment:
All other `build.gradle` files for the other plugins use :
`compileSdk 34`
Change it to match that as well.

Reply:
Okay